cli/js/fetch_test.ts

```diff
@@ -5,9 +5,12 @@ import {
   assert,
   assertEquals,
   assertStrContains,
-  assertThrows
+  assertThrows,
+  fail
 } from "./test_util.ts";
 
+import { Response } from "./fetch.ts";
```

Comment:
`Response` is available as global, you needn't import it - that causes the tests to fail

Reply:
That said - I see you made changes to `Response` - you'll need to update corresponding type declaration in `lib.deno_runtime.d.ts`